src/adapters/picks/picks.ts

```diff
@@ -0,0 +1,13 @@
+import { DBGetFilteredPicksWithCount } from "../../ports/picks"
+import { PickUserAddressesWithCount } from "./types"
+
+export function fromDBGetPickByItemIdToPickUserAddressesWithCount(
+  dBGetPicksByListId: DBGetFilteredPicksWithCount[]
+): PickUserAddressesWithCount {
+  return {
+    results: dBGetPicksByListId.map((pick) => ({
```

Comment:
As the other adapters don't use the `result` key for the picks, but the `picks` key, what do you think about using `picks` as well?
```suggestion
    picks: dBGetPicksByListId.map((pick) => ({
```